misc/PFS_Abbreviations.tsv

```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
+iic	ICS	Actor	Instrument Interface and Controller	ICS software module which organizes operation of PFS subsystems in an exposure sequence, in harmony with the telescope.
+IIS	SpS	Subcomponent	Internal Illumination Source	Light source of SpS to illuminate engineering fibers.
+IPMU	General	Organization	Kavli Institute for the Physics and Mathematics of the Universe, the University of Tokyo	PFS collaborator (PI) taking initiative to proceed the project. IPMU also has a responsibility to develop software interfacing between PFS and Subaru, and execute system commissioning.
+IR3	Subaru	Facility	IR-side 3rd floor in the dome building 	The place on which the UPS, chillers for spectrograph modules and the spectrograph clean room (SCR) will be accommodated.
+IR4	Subaru	Facility	IR-side 4th floor in the dome building 	The place on which the spectrograph clean room (SCR) will be constructed.
 JHU	General	Organization	Johns Hopkins University, USA	PFS collaborator. JHU has a responsibility for development of Spectrographs System.
 JPL	General	Organization	Jet Propulsion Laboratory, USA	PFS collaborator. JPL has a responsibility for control system of Cobra positioners.
 LAM	General	Organization	Laboratoire d'Astrophysique de Marseille, France	PFS collaborator. LAM leads development of Spectrographs Systems (SpS), and has a responsibility for development of 1D-data reduction pipeline (DRP).
 LCU	SpS	Subcomponent	Local Control Unit	Control system for Spectrograph Module (SM). Power  distribution units, network device and controllers are accommodated to dedicated rack. Each SM has one LCU. LAM, JHU and PU have a responsibility for its development.
 LNA	General	Organization	Laborato\'rio Nacional de Astrofi\'sica, Brazil	PFS collaborator. LNA has a responsibility for development of Fiber Optical Cable and Connector System (FOCCoS).
 MAC	ICS	General	MLP1 AG Converter	Software as an interface between PFS Auto Guide Cameras (AGCs) and Subaru Mid-Level Processors 1 (MLP1). MAC calculates tracking error using information from AGCs and send them to MLP1. MAC also sends AGCs image to guide viewer after converting proper format. IPMU has a responsibility for its development.
 MCPS	PFI	Subcomponent	Master Controller Power Supply	Device of Positioner Remote Electronics (PRE). MCPS supplies DC power to controller modules of Cobra positioners.
-MCS	PFS	Subsystem	Metrology Camera System	PFS instrument subsystem installed to the Cassegrain focus. MCS takes image of back-illuminated fibers, and measures their centroids. ASIAA has a resiponsibility for ites development.
+MCS	PFS	Subsystem	Metrology Camera System	PFS instrument subsystem installed to the Cassegrain focus. MCS takes image of back-illuminated fibers, and measures their centroids. ASIAA has a responsibility for its development.
+meb	ICS	Actor	MCS Electronic Box interface	Software to control power supply and telemetry system for MCS. ASIAA has a responsibility for its development.
```

Comment:
also hardware definition exists??

Reply:
Thank you for suggestion. As I just followed product tree, I missed it. I'll add it.